packages/js/src/classic-editor/decorators/marks.js

```diff
@@ -0,0 +1,75 @@
+import { useDispatch, useSelect } from "@wordpress/data";
+import { useEffect } from "@wordpress/element";
+import { MARKER_STATUS, SEO_STORE_NAME } from "@yoast/seo-integration";
+import { forEach } from "lodash";
+import { markers } from "yoastseo";
+import useEditor from "../hooks/useEditor";
+
+/**
+ * @param  {string} target eew.
+ * @param  {string} name eew.
+ * @param  {object} descriptor eew.
+ *
+ * @returns {void}
+ */
+const marksDecorator = () => {
```

Comment:
This looks like a hook, so the name should start with `use`. `useMarksDecorator` maybe? Shouldn't it also live in the hooks folder?

Reply:
I think this is a hook indeed, moved it. And split up into `useMarker` and `useMarkerStatus`, since they do their own thing.